src/quantum/azext_quantum/operations/job.py

```diff
@@ -265,7 +270,7 @@ def output(cmd, job_id, resource_group_name=None, workspace_name=None, location=
         if len(lines) == 0:
             return
 
-        if job.target.startswith("microsoft.simulator"):
+        if job.target.startswith("microsoft.simulator") and job.target != "microsoft.simulator.resources-estimator":
```

Comment:
Is this the correct name?

Reply:
yes I think we changed the sku name, but not the target name